ledger/src/shred.rs

```diff
@@ -263,20 +263,18 @@ impl Shred {
     }
 
     pub fn new_from_serialized_shred(mut payload: Vec<u8>) -> Result<Self> {
+        // A shred can be deserialized in several cases; payload length will vary for these:
+        //   payload.len() <= SHRED_PAYLOAD_SIZE when payload is retrieved from the blockstore
+        //   payload.len() == SHRED_PAYLOAD_SIZE when payload is from a local shred (shred.payload)
+        //   payload.len()  > PACKET_DATA_SIZE when payload comes from a packet (window serivce)
+        // Resize here so the shreds always have the same length
+        payload.resize(SHRED_PAYLOAD_SIZE, 0);
```

Comment:
@jeffwashington - For the sake of more datapoints on this, did you encounter any issues after applying this change locally? This was the change to allow use of older stack (such as v1.6) on a ledger populated by master

Reply:
From DM with Jeff:
> so, you’re trying to get more data points on your change on shreds?
> I have not run into any issues.